terra-jupyter-base/Dockerfile

```diff
@@ -98,29 +99,10 @@ RUN useradd -m -s /bin/bash -N -u $WELDER_UID $WELDER_USER
 ENV JUPYTER_PORT 8000
 ENV JUPYTER_HOME /etc/jupyter
 
-ENV PYSPARK_DRIVER_PYTHON jupyter
-ENV PYSPARK_DRIVER_PYTHON_OPTS notebook
-
-ENV SPARK_HOME=/usr/lib/spark
-ENV PATH $SPARK_HOME:$SPARK_HOME/python:$SPARK_HOME/bin:$HAIL_HOME:$PATH
-ENV PYTHONPATH $PYTHONPATH:$HAIL_HOME/$HAILPYTHON:$HAIL_HOME/python:$SPARK_HOME/python:$JUPYTER_HOME/custom
-
-# Python2, kernel, and pip
-RUN apt-get update \
- && apt-get install -yq --no-install-recommends \
-    python \
-    python-dev \
-
- # jessie's default pip doesn't work well with jupyter
- && wget -nv https://bootstrap.pypa.io/get-pip.py \
- && python get-pip.py \
- && pip install ipykernel==4.10.0 \
- && python2 -m ipykernel install --name python2 --display-name "Python 2" \
- && apt-get clean \
- && rm -rf /var/lib/apt/lists/*
+ENV PYTHONPATH $PYTHONPATH:python:$JUPYTER_HOME/custom
 
 # Python3, kernel, and pip
-ENV PYTHON_VERSION 3.6.8
+ENV PYTHON_VERSION 3.7.0
 
 # lifted from https://github.com/docker-library/python/blob/dd36c08c1f94083476a8579b8bf20c4cd46c6400/3.6/stretch/Dockerfile
```

Comment:
Just wondering, should we avoid building python from source? We only did that because 3.6.8 wasn't in debian's repo. A 3.7 binary should be available, I think. 

However it doesn't hurt anything to build from source, so I could go either way.